### PR TITLE
Add Redlink docs

### PR DESCRIPTION
**Pull request by plotkabytes**
Documentation for new redlink notifier. PR: [symfony/symfony#50170](https://github.com/symfony/symfony/pull/50170).

### CODE REVIEW

File: notifier.rst [hunk 1]
```diff
@@ -112,7 +113,7 @@ Service             Package                                DSN
 
 .. versionadded:: 6.3
 
-    The Bandwith, iSendPro, Plivo, RingCentral, SimpleTextin and Termii integrations
+    The Bandwith, iSendPro, Plivo, RingCentral, SimpleTextin, Termii and Redlink integrations
     were introduced in Symfony 6.3.
```
Comment (OskarStark) [inline notifier.rst]: Please create a new block for versionadded:: 6.4, because 6.3 is already in feature freeze, thanks

Comment (m-zyla) [reply notifier.rst]: Done

Comment (plotkabytes) [reply notifier.rst]: Done

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Please target 6.4 branch

**Comment by OskarStark** [comment]
Thanks for your work on this new feature and congratulations on your first contribution to the Symfony documentation 🎉

**Pull request merged**